account_payment_widget_amount/__manifest__.py

```diff
@@ -1,18 +1,18 @@
 # Copyright 2019-2021 ForgeFlow S.L.
+# Copyright 2024 OERP Canada <https://www.oerp.ca>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "Account Payment Widget Amount",
     "summary": "Extends the payment widget to be able to choose the payment " "amount",
-    "version": "16.0.1.0.0",
+    "version": "17.0.1.0.0",
     "category": "Account-payment",
     "website": "https://github.com/OCA/account-payment",
-    "author": "ForgeFlow S.L., Odoo Community Association (OCA)",
+    "author": "ForgeFlow S.L., OERP Canada, Odoo Community Association (OCA)",
```

Comment:
You should not add yourself to authors. More contribution should be added to the module in order to do that.